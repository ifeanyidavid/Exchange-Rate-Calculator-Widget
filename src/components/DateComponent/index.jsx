import React from "react";
import styled from "styled-components";

const DateWrapper = styled.div`
  display: flex;
  justify-content: center;
  margin-bottom: 1rem;

  .text-muted {
    color: rgba(0, 0, 0, 0.5);
  }
`;

function DateComponent() {
  return (
    <DateWrapper>
      <span className="text-muted">{new Date().toDateString()}</span>
    </DateWrapper>
  );
}

export default DateComponent;
