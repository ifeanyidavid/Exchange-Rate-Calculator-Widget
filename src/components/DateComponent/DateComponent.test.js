import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJson from "enzyme-to-json";
import DateComponent from ".";

configure({ adapter: new Adapter() });

describe("DateComponent component", () => {
  it("Renders correctly", () => {
    // wrapper instance around rendered output
    const wrapper = shallow(<DateComponent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
