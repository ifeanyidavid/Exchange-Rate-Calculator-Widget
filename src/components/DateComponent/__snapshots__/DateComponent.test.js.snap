// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateComponent component Renders correctly 1`] = `
<styled.div>
  <span
    className="text-muted"
  >
    Tue Dec 10 2019
  </span>
</styled.div>
`;
